CHANGELOG.md

```diff
@@ -7,10 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
-- Moved `opcounters`, `opcountersRepl` to `common_collector` (#146), thx [@nikita-b](https://github.com/nikita-b). 
 - [PMM-3511](https://jira.percona.com/browse/PMM-3511): Switched to [mongo-go-driver](https://github.com/mongodb/mongo-go-driver).
-- [PMM-4168](https://jira.percona.com/browse/PMM-4168): Updated all dependencies including `github.com/prometheus/golang_client`. 
-As a result a lot of metrics was changed from Counter to ConstMetric. For more information look at commit [b8e8369](https://github.com/percona/mongodb_exporter/pull/145/commits/b8e836973e1e12608c24c0fe01437dfc49f88ae3).
```

Comment:
The `Counter` is a metric type, that is important to users. `ConstMetric` is an implementation detail that is not.
Originally, that entry was about Counter => Gauge changes, but they were reverted.